modules/power-monitoring-configuring-kepler-redfish.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/configuring-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-configuring-kepler-redfish_{context}"]
+= Configuring {PM-kepler} to use Redfish
+
+You can configure the {PM-kepler} to use the Redfish as the source for platform power.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*. If you already have a {PM-kepler} instance created, click *Edit Kepler*.
+
+. Configure {PM-kepler} instance’s `.spec.exporter.redfish` specifying the mandatory `secretRef` and optional `probeInterval` and `skipSSLVerify`.
++
+.Example output
+[source,yaml]
+----
+apiVersion: kepler.system.sustainable.computing.io/v1alpha1
+kind: Kepler
+metadata:
+  name: kepler
+spec:
+  exporter:
+    deployment:  ... 
+    redfish:
+      secretRef: <name of the secret> required <1>
+      probeInterval: 60s <2>
+      skipSSLVerify: false <3>
+----
+<1> Specifies the name of the secret that contains the credentials for accessing the Redfish server. This field is required.
+<2> Controls the frequency at which the power information is queried from Redfish. This field is optional. The default value is `60s`.
+<3> Controls if {PM-kepler} skips verifying the Redfish server certificate. This field is optional. The default value is `false`.
++
+[NOTE]
+====
+The `openshift-power-monitoring` namespace is created after the {PM-kepler} is deployed.
+====
+. Create the `redfish.csv` file with this data format `your_kubelet_node_name, redfish_username, redfish_password, https://redfish_ip_or_hostname/`.
++
+.Example output
+[source,csv]
+----
+control-plane,dummyuser,dummypass,https://redfish.nodes.example.com
+worker-1,dummyuser,dummypass,https://redfish.nodes.example.com
+worker-2,dummyuser,dummypass,https://another.redfish.nodes.example.com
+----
+. Create the secret under `openshift-power-monitoring` namespace. You must create the secret with the following conditions:
+* The secret type is `Opaque`
+* The credentials are stored under the `redfish.csv` key in the `data` field of the secret.
++
+[source,terminal]
+----
+$ oc -n openshift-power-monitoring \
+  create secret generic redfish-secret \
+  --from-file=redfish.csv
+----
++
+.Example output
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: redfish-scret
+data:
+  redfish.csv: YmFyCg==
+----
+[IMPORTANT]
+====
+The {PM-kepler} deployment will not continue until the Redfish secret is created. You can find this information in the status of a {PM-kepler} instance.
+====
```

Comment:
The rendering seems off. Are we missing a `+` ?

Reply:
Yes. fixing this. Thank you!